packages/ses/src/tame-function-constructors.js

```diff
@@ -126,11 +126,18 @@ export default function tameFunctionConstructors() {
     '%InertAsyncFunction%',
     '(async function(){})',
   );
-  repairFunction(
-    'AsyncGeneratorFunction',
-    '%InertAsyncGeneratorFunction%',
-    '(async function*(){})',
-  );
+
+  try {
+    // eslint-disable-next-line
+    new FERAL_FUNCTION('async function* (){}');
+    repairFunction(
+      'AsyncGeneratorFunction',
+      '%InertAsyncGeneratorFunction%',
+      '(async function*(){})',
+    );
+  } catch {
+    // 
+  }
```

Comment:
removing [prev](https://github.com/endojs/endo/pull/2334/files#r1654976413) try/catch and this one fixes tests

Reply:
fixed in https://github.com/endojs/endo/pull/2334/commits/74a580c1acd6010c79323cfcce539bfac109a032